lib/library.go

```diff
@@ -327,6 +327,14 @@ func Login(address, password *C.char) *C.char {
 	return makeJSONResponse(err)
 }
 
+// LoginWithKeycard initializes an account with a chat key and encryption key used for PFS.
+// It purges all the previous identities from Whisper, and injects the key as shh identity.
```

Comment:
Haven't seen the code fragment that purges all previous identities from Whisper.

Reply:
@adambabik I used some part of the comment from the `Login` function to show it does the same. Basically we call `whisperService.SelectKeyPair`, which reset the whisper key store and inject the new key.